<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleTrainArea</class>
 <widget class="QWidget" name="VehicleTrainArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(0,0,50);
}
QLabel
{

font:24px,&quot;微软雅黑&quot;;
border:1px solid rgb(0,0,0);

}
</string>
  </property>
  <widget class="QLabel" name="lbl_door4_5">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door4_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door4_6">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_1">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door4_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door4_1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door2_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door1_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door3_3">
   <property name="geometry">
    <rect>
     <x>608</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_door4_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_quickBreak1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_quickBreak2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_quickBreak3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmp1">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>185</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmp2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>185</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmp3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>185</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmp4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>185</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmpToken1">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/tmptoken.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_panto1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pantoNormalDown.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_panto2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/pantoNormalDown.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmpToken2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/tmptoken.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmpToken3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/tmptoken.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tmpToken4">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>187</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/tmptoken.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_direction">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>114</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowleft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_active">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>115</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_direction">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>114</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowright.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_active">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>115</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_C_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>112</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);
border:0px solid black;
font: 15pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>1车</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_C_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>112</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);
border:0px solid black;
font: 15pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2车</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_C_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>112</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);
border:0px solid black;
font: 15pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>3车</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_C_9">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>112</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 170);
border:0px solid black;
font: 15pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>4车</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metrogou1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>114</y>
     <width>10</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metrogou2">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>114</y>
     <width>10</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-left-radius:8px;
border-bottom-left-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-right-radius:8px;
border-bottom-right-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroPower1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:14px &quot;微软雅黑&quot;;
background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);
border:2px solid black;</string>
   </property>
   <property name="text">
    <string>PH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroPower2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:14px &quot;微软雅黑&quot;;
background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);
border:2px solid black;</string>
   </property>
   <property name="text">
    <string>PH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_quickBreak4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>110</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>LBLM2</zorder>
  <zorder>LBLM1</zorder>
  <zorder>LBLMP1</zorder>
  <zorder>LBLTC1</zorder>
  <zorder>lbl_door4_5</zorder>
  <zorder>lbl_door2_5</zorder>
  <zorder>lbl_door4_4</zorder>
  <zorder>lbl_door3_4</zorder>
  <zorder>lbl_door3_1</zorder>
  <zorder>lbl_door4_6</zorder>
  <zorder>lbl_door3_2</zorder>
  <zorder>lbl_door2_1</zorder>
  <zorder>lbl_door4_3</zorder>
  <zorder>lbl_door2_4</zorder>
  <zorder>lbl_door4_1</zorder>
  <zorder>lbl_door1_4</zorder>
  <zorder>lbl_door2_6</zorder>
  <zorder>lbl_door1_6</zorder>
  <zorder>lbl_door1_1</zorder>
  <zorder>lbl_door3_6</zorder>
  <zorder>lbl_door2_3</zorder>
  <zorder>lbl_door3_5</zorder>
  <zorder>lbl_door2_2</zorder>
  <zorder>lbl_door1_3</zorder>
  <zorder>lbl_door1_2</zorder>
  <zorder>lbl_door1_5</zorder>
  <zorder>lbl_door3_3</zorder>
  <zorder>lbl_door4_2</zorder>
  <zorder>lbl_quickBreak1</zorder>
  <zorder>lbl_quickBreak2</zorder>
  <zorder>lbl_quickBreak3</zorder>
  <zorder>lbl_tmp1</zorder>
  <zorder>lbl_tmp2</zorder>
  <zorder>lbl_tmp3</zorder>
  <zorder>lbl_tmp4</zorder>
  <zorder>lbl_tmpToken1</zorder>
  <zorder>lbl_panto1</zorder>
  <zorder>lbl_panto2</zorder>
  <zorder>lbl_tmpToken2</zorder>
  <zorder>lbl_tmpToken3</zorder>
  <zorder>lbl_tmpToken4</zorder>
  <zorder>lbl_cab1_direction</zorder>
  <zorder>lbl_cab1_active</zorder>
  <zorder>lbl_cab2_direction</zorder>
  <zorder>lbl_cab2_active</zorder>
  <zorder>lbl_C_1</zorder>
  <zorder>lbl_C_7</zorder>
  <zorder>lbl_C_8</zorder>
  <zorder>lbl_C_9</zorder>
  <zorder>lbl_metrogou1</zorder>
  <zorder>lbl_metrogou2</zorder>
  <zorder>lbl_metroPower1</zorder>
  <zorder>lbl_metroPower2</zorder>
  <zorder>lbl_quickBreak4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
